sklearn/dummy.py

```diff
@@ -580,13 +580,8 @@ def score(self, X, y, sample_weight=None):
 
         Parameters
         ----------
-        X : {array-like, None}
-            Test samples with shape = (n_samples, n_features) or None.
-            For some estimators this may be a
-            precomputed kernel matrix instead, shape = (n_samples,
-            n_samples_fitted], where n_samples_fitted is the number of
-            samples used in the fitting for the estimator.
```

Comment:
I do not know why this paragraph was here. It should be similar to the `score` method in `sklearn.dummy.DummyClassifier`.